ee/ucp/admin/configure/join-nodes/join-windows-nodes-to-cluster.md

```diff
@@ -6,23 +6,25 @@ redirect_from:
   - /datacenter/ucp/3.0/guides/admin/configure/join-nodes/join-windows-nodes-to-cluster/
 ---
 
-Docker Enterprise 2.1 supports worker nodes that run on Windows Server 2016, 1709, 
-or 1803. Only worker nodes are supported on Windows, and all manager nodes in the cluster
-must run on Linux.
+Docker Enterprise 3.0 supports worker nodes that run on Windows Server 2016 and
+Windows Server 2019. Only worker nodes are supported on Windows, and all
+manager nodes in the cluster must run on Linux. Additionally Windows worker
+nodes can only be used the Swarm Orchestrator.
```

Comment:
this sentence is missing a word. Should it read "can only be used *by* the Swarm Orchestrator"?